kubernetes/zenko/templates/cosmos-operator/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "cosmos-operator.fullname" . }}
+  labels:
+    app: {{ template "cosmos-operator.name" . }}
+    chart: {{ template "zenko.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "cosmos-operator.name" . }}
+      name: {{ template "cosmos-operator.fullname" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "cosmos-operator.name" . }}
+        name: {{ template "cosmos-operator.fullname" . }}
+        release: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ template "cosmos-operator.fullname" . }}
+      containers:
+        - name: {{ template "cosmos-operator.fullname" . }}
+          image: "{{ .Values.cosmos.operator.image.repository }}:{{ .Values.cosmos.operator.image.tag }}"
+          imagePullPolicy: {{ .Values.cosmos.operator.image.pullPolicy }}
+          ports:
+          - containerPort: 60000
+            name: metrics
+          env:
+            - name: WATCH_NAMESPACE
```

Comment:
If the operator runs in a specific namespace (which is a good initial approach!), why does it get PVC access rights through a ClusterRole(Binding)?

Reply:
It does need to. Removing it...